tests/PHPUnit/proxy/misc

```diff
@@ -0,0 +1 @@
+/home/vagrant/www/piwik/misc
```

Comment:
This file probably shouldn't be added, just a symlink (probably should be in .gitignore).

Reply:
yes definitely 👍  no idea how it ended up there.

Reply:
also adding it to gitignore